bci-ssvep/src/process/cca_np_fir.py

```diff
@@ -17,18 +17,84 @@
 logging.config.dictConfig(log_init.init_log_cfg(__file__))
 log = logging.getLogger(__name__)
 
-
 np_base_url = 'http://127.0.0.1:6937'
 pipeline_file = os.path.abspath(r'../pipelines/SSVEP Sequential.pyp')
 max_threads = os.cpu_count()
 
-
 database_url = 'sqlite:///../../database/data/ssvep-db.sqlite'
 database_recordings_folder = os.path.abspath(r'../../database/data/recordings')
 database_temp_folder = os.path.abspath(r'../../database/data/temp')
 
 engine = create_engine(database_url)
 
+import numpy as np
+from sklearn.feature_selection import mutual_info_regression
+from scipy.optimize import minimize
+
+
+class ITCCA:
```

Comment:
Move class to another file and maybe make it not a class but just a function in another class